docs/operator-manual/upgrading/3.2-3.3.md

```diff
@@ -2,6 +2,31 @@
 
 ## Breaking Changes
 
+### ApplicationSet CRD exceeds the size limit for client-side apply
+
+In this version, the ApplicationSet CRD exceeds the size limit for client-side apply, meaning that the previous upgrade methods will result in `The CustomResourceDefinition "applicationsets.argoproj.io" is invalid: metadata.annotations: Too long: may not be more than 262144 bytes`
+
+In order to avoid this issue, it is now required to upgrade Argo CD using Server Side Apply (SSA) along with `resolve conflicts`.
```

Comment:
```suggestion
To avoid this issue, Argo CD must now be upgraded using Server-Side Apply (SSA).

When using SSA for upgrades, conflict resolution must be enabled. The `--force-conflicts` flag allows the upgrade to take ownership of fields that may have been previously managed by other tools (for example, Helm or a prior `kubectl apply`).
```